packages/typespec-go/test/cadlranch/payload/multipartgroup/zz_formdata_client.go

```diff
@@ -20,7 +21,7 @@ type FormDataClient struct {
 
 // Basic - Test content-type: multipart/form-data
 //   - options - FormDataClientBasicOptions contains the optional parameters for the FormDataClient.Basic method.
-func (client *FormDataClient) Basic(ctx context.Context, body MultiPartRequest, options *FormDataClientBasicOptions) (FormDataClientBasicResponse, error) {
+func (client *FormDataClient) Basic(ctx context.Context, body io.ReadSeekCloser, options *FormDataClientBasicOptions) (FormDataClientBasicResponse, error) {
```

Comment:
This is incorrect and will be fixed in a separate PR for multipart/form support.